actions/core/build.gradle.kts

```diff
@@ -41,6 +41,7 @@ dependencies {
     implementation(group = "org.apache.commons", name = "commons-lang3")
     implementation(group = "com.google.guava", name = "guava")
 
+    testImplementation("io.knotx:knotx-junit5:${project.version}")
```

Comment:
Can we add junit5 dependency in Knot.x Gradle plugin (tests)?

Reply:
I removed it intentionally [here](https://github.com/Knotx/knotx-gradle-plugins/pull/10/files#diff-2d975115757706613ad40a2a88a38a05L40).
It caused cyclic dependency plugins needed jUnit to have depndency, but jUnit uses plugin to do some tests: https://github.com/Knotx/knotx-junit5/blob/master/build.gradle.kts#L25